items/weapons.md

```diff
@@ -0,0 +1,51 @@
+# Weapons
+Weapons are items that can be equipped
+
+## General Template
+As well as the base items [General Template](README.md#general-template), weapons define these:
+
+### Dice
+The dice that are rolled for an attack made with this. See [Attacks](../systems/rolls.md#attacks)
+
+### Damage Type
+The type of damage this weapon deals by default. See [Damage Types](../systems/combat.md#damage-types)
+
+### Traits
+A list of [weapon traits](#weapon-traits) this weapon has.
+
+### Range
+The maximum effective range on this weapon.
+
+# Melee weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Sword          |A sword                                                      |2      |25     |2d6    |Slashing     |1-handed                       |5ft    |
+|War-hammer     |A wooden shafted hammer with a spike on one side             |2      |25     |2d6    |Bludgeoning  |1-handed                       |5ft    |
+|Spear          |A long wooden shaft with a metal point on the end            |2      |25     |2d6    |Piercing     |1-handed, versatile, throwing  |5ft    |
+|War-axe        |An axe with a wide head                                      |2      |25     |2d6    |Piercing     |1-handed                       |5ft    |
+
+# Ranged weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Short bow      |A short bow                                                  |2      |25     |2d6    |Piercing     |2-handed                       |120ft  |
+|Crossbow       |A crossbow                                                   |2      |25     |2d6    |Piercing     |2-handed, loading              |120ft  |
+
+# Weapon Traits
+
+## 1-handed
+This weapon can be wielded with only one hand
+
+## 2-handed
+This weapon requires two hands to wield
+
+## Versatile
+This weapon can be wielded with a second hand to increase its dice
+
+## Throwing
+This weapon can be thrown without [disadvantage](../systems/rolls.md#disadvantage). When thrown, [Range](#range) is 20ft.
```

Comment:
Can a great axe be thrown for say.. 2d10 -1d4 damage? I wonder if a two handed weapon should be double disadvantage when thrown.
This may be too complicated.

Reply:
I guess if it is out of range (5ft) does it always fail? Or maybe we stack more disadvantages on it.

Reply:
Well, I do really like being able to throw other weapons.